frontend/src/Reviews.js

```diff
@@ -1,57 +1,106 @@
+                );
 
-            if (response && response.status === 200) {
-                const newVal = food[type] + 1;
+                if (response && response.status === 200) {
+                    const newVal = food[type] + 1;
 
-                if (type === "likes") {
-                    setFood({
-                        ...food,
-                        likes: newVal,
-                    });
-                } else if (type === "dislikes") {
-                    setFood({
-                        ...food,
-                        dislikes: newVal,
-                    });
-                } else {
-                    setFood({
-                        ...food,
-                        poisonings: newVal,
-                    });
+                    if (type === "likes") {
+                        setLFlag("True");
+                        setFood({
+                            ...food,
+                            likes: newVal,
+                        });
+                    } else if (type === "dislikes") {
+                        setDFlag("True");
+                        setFood({
+                            ...food,
+                            dislikes: newVal,
+                        });
+                    } else {
+                        setPFlag("True");
+                        setFood({
+                            ...food,
+                            poisonings: newVal,
+                        });
+                    }
                 }
+            } catch (error) {
+                //We're not handling errors. Just logging into the console.
+                console.error(error);
             }
-        } catch (error) {
-            //We're not handling errors. Just logging into the console.
-            console.error(error);
-        }
+        }    
     };
 
-    const handleSubmit = async (event) => {
-        event.preventDefault();
-        console.log("sending reivew", review);
+    async function getReviewsById() {
         try {
-            const result = await axios.post(
-                `http://localhost:4000/foods/${food._id}/reviews`,
-                { review }
+            const response = await axios.get(
+                `http://localhost:4000/foods/${id}/reviews`,
             );
+            setReviews(reviews);
+            return response;
         } catch (error) {
+            //We're not handling errors. Just logging into the console.
             console.log(error);
+            return false;
         }
-    };
-    const handleChange = (event) => {
-        setReview(event.target.value);
+    }
+
+    const handleSubmit = async (event) => {
+        if(rFlag === "True"){
+            alert("Cannot Add Another Review");
+            event.preventDefault();
+        }
+        else {
+            const review = document.getElementById("reviewInputField").value;
+            setRFlag("True");
+            event.preventDefault();
```

Comment:
event.preventDefault() could be moved to the top of the function so it doesn't need to be stated in each branch